x-pack/plugins/security_solution/public/app/deep_links/index.ts

```diff
@@ -349,7 +350,7 @@ export function getDeepLinks(
       (deepLink) =>
         (deepLink.id !== SecurityPageName.case && deepLink.id !== SecurityPageName.ueba) || // is not cases or ueba
         (deepLink.id === SecurityPageName.case &&
-          (capabilities == null || capabilities.siem.read_cases === true)) || // is cases with at least read only caps
+          (capabilities == null || capabilities[CASES_FEATURE_ID].read_cases === true)) || // is cases with at least read only caps
```

Comment:
I have a conflicting PR https://github.com/elastic/kibana/pull/113046 but no worries I will sync mine when this one is merged